activiti-services/activiti-services-core/src/main/java/org/activiti/services/core/commands/StartProcessInstanceCmdExecutor.java

```diff
@@ -26,8 +28,10 @@ public Class getHandledType() {
     }
 
     @Override
-    public void execute(Command cmd) {
-        ProcessInstance processInstance = processEngine.startProcess((StartProcessInstanceCmd) cmd);
-        System.out.println(processInstance.getId() + " -  " + processInstance);
+    public void execute(StartProcessInstanceCmd cmd) {
+        ProcessInstance processInstance = processEngine.startProcess(cmd);
+        StartProcessInstanceResults cmdResult = new StartProcessInstanceResults(cmd.getId(),
```

Comment:
Are we expecting to have the same content here as the one we get by calling the REST end point? In this case, we should include all the process instance, not only the id. This applies to all `*Results`.